testing/regress/ecl/cppbody11.ecl

```diff
@@ -24,7 +24,7 @@ real scoreIt(scoreFunc func, real a, real b) := DEFINE
     
 real scoreIt2(scoreFunc func, real a, real b) := BEGINC++
     
-    return func(a-1.0,b-1.0) * func(a+1.0,b+1.0);
```

Comment:
(I think you've made it default to passing the context to inline - which is fine, but implies you did NOT need the #option context in the next example)